drivers/storage/class/classpnp/power.c

```diff
@@ -1622,7 +1622,7 @@ ClasspPowerHandler(
                     if (fdoData->HotplugInfo.MediaRemovable || fdoData->HotplugInfo.MediaHotplug) {
                         /*
                             *  We are suspending device and this drive is either hot-pluggable
-                            *  or contains removeable media.
+                            *  or contains removable media.
```

Comment:
This is 3rd-party code, so please revert.
https://github.com/reactos/reactos/blob/719ea022ec5d85a41c3df3d79ed22fe1f93c2db8/media/doc/3rd%20Party%20Files.txt#L135-L139

Reply:
I've removed the scope creep and committed only the intended changes.